src/routes/Homepage/Homepage.scss

```
@@ -31,26 +31,18 @@
 }
 
 .homepage__language {
-  background: $color-lighter-gray;
-}
-
-html:not([theme="dark"]) {
-  .homepage__language {
-    box-shadow: 0 0 0 0 rgba($color-middle-gray, 0) !important;
-  }
+  background: $color-light-gray;
+  box-shadow: 0 0 0 0 rgba($color-middle-gray, 0);
```

```suggestion
  box-shadow: 0 0 rgba($color-middle-gray, 0);
```
`blur-radius` and `spread-radius` default to 0, so the values can be omitted